### ISSUE TITLE
The metadata parsing failed, making sure you use an official YOLOv8 model

### ISSUE THREAD

**Issue by PhamZung26** [dme-compunet/YoloV8#86]
I have a self-trained model, and when loading the model, I encountered the following error
```
System.InvalidOperationException
  HResult=0x80131509
  Message=The metadata parsing failed, making sure you use an official YOLOv8 model
  Source=YoloV8
  StackTrace:
   at Compunet.YoloV8.Metadata.YoloMetadata.Parse(InferenceSession session)
   at Compunet.YoloV8.Services.Resolvers.PredictorServiceResolver..ctor(InferenceSession session, YoloConfiguration configuration)
   at Compunet.YoloV8.YoloPredictor..ctor(Byte[] model, YoloPredictorOptions options)
   at Compunet.YoloV8.YoloPredictor..ctor(String path)
   at Program.<Main>$(String[] args) in C:\Users\Admin\source\repos\TestYoLoNet8\TestYoLoNet8\Program.cs:line 4

  This exception was originally thrown at this call stack:
    [External Code]

Inner Exception 1:
NotSupportedException: Unrecognized YOLO model architecture

```
This is my model and meta data

[DetectContainerNoV8.zip](https://github.com/user-attachments/files/17372946/DetectContainerNoV8.zip)
![image](https://github.com/user-attachments/assets/df0f4823-a797-497a-a49e-7791006c659b)

**Comment by dme-compunet** [comment]
Thanks for the report, can you check if this issue is resolved in version `5.1.1`?

**Comment by PhamZung26** [comment]
I can't install version 5.1.1 on .NET 8

**Comment by dme-compunet** [comment]
The problem should be solved in the latest version (`5.2.0`), if so please close the issue
